NSS-Day-Cohort-68/rare-client-rare-team-2
# Feature/create comment

There is no current button that links to the Add Comment page because there is not currently a Post Details page.

Added comments component that displays a form for users to fill out if they want to add a comment to a post.
Once the user has added text, the user may press send to save it to the database.

TESTING:
1. Make sure API is running and Comments table is added with a "created_on" column
2. Navigate to http://localhost:3000/posts/1/comments in your browser
3. Enter a comment
4. Click Send
5. Check database to see if the new comment is present
